plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/payment-methods-selection/style.scss

```diff
@@ -138,8 +132,13 @@
 		padding: 0 !important; // There is a padding on the parent container that we need to override.
 		border-bottom: none !important;
 		position: fixed;
+		// Positioning the container to the right side of the screen.
+		right: 16px;
+		left: 296px; // Account for the sidebar width + the gap between the sidebar and the content.
 
-		@media screen and (max-width: $break-medium) {
+		@media screen and (max-width: 768px) {
```

Comment:
The hardcoded breakpoint is used on purpose to avoid layout issues that occur when the window width falls between 768px and 780px.